docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -147,7 +147,7 @@ databases [in this guide][link-connecting-to-db].
 | Environment variable            | Description                                                                                                                             | Possible Values                                             |
 | ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
 | `CUBESTORE_BIND_ADDR`           | The address/port pair for Cube Store's MySQL-compatible interface. Defaults to `0.0.0.0:3306`                                           | A valid address/port pair                                   |
-| `CUBESTORE_DATA_DIR`            | A path on the local filesystem to store a local replica of the data. Defaults to `.cubestore/data`                                      | A valid path on the local filesystem with read/write access |
+| `CUBESTORE_DATA_DIR`            | A path on the local filesystem to store a local replica of the data. Must be unique to each node. Defaults to `.cubestore/data`         | A valid path on the local filesystem with read/write access |
```

Comment:
I think it's better to say "...unique on each node"?

Also, not sure if it's helpful to expand by noting that "...for example `data_dir` should be different from `remote_dir`" You can ignore this if you think it's clear enough for most people :-)

Reply:
@rpaik, thank you for the correct remark! 
You are right about `remote_dir`. I added info about this case.

Could you please check again?

Reply:
Looks good to me. Thanks @rchkv 😄